### DISCUSSION TITLE
Does this extension work on Chrome OS in Chrome browser?

### DISCUSSION THREAD

**Discussion by poythres** [JohnnyRS/PandaCrazy-Max#53] [Q&A]
I installed the extension on my Chromebook but when I open it, the overlay gets stuck at "Please wait. Loading up all data for you." None of the other loading messages appear below it that typically do on my PC.

Is this a bug I can workaround somehow or is it expected this extension doesn't support Chrome OS?

Thanks,

**Comment by JohnnyRS** [comment]
There could be some changes with the chrome browser on chrome OS and other operating systems that may be causing that. I haven't been able to test in under chrome OS so can't guarantee if it will work or not. But what you described happened to me when trying to get it to work in firefox and was able to fix it so maybe that may fix it with chrome OS too. I'm finishing up the next update and soon will add it on here and then the chrome store. You can try it again with the next update and report back if it worked. Today or tomorrow I am hoping to have the next update ready.
